tools/llm_bench/llm_bench_utils/ov_utils.py

```diff
@@ -603,10 +597,8 @@ def create_genai_image_text_gen_model(model_path, device, ov_config, memory_moni
 
     processor_config = get_vlm_processor(model_path)
 
-    cb = ov_config.get('ATTENTION_BACKEND', '') == PA_ATTENTION_BACKEND
     cb_config = kwargs.get("cb_config")
-    if cb or cb_config is not None:
-        log.info("Continuous Batching mode activated")
+    if cb_config is not None:
```

Comment:
What if we would like to specify `enable_prefix_cache=False` via cb_config in llm_bench?

Reply:
Or we can use `--disable_prompt_permutation` option 
https://github.com/openvinotoolkit/openvino.genai/blob/23bf529766c52a05de074fd53637c321271d821b/tools/llm_bench/benchmark.py#L192

in `visual_language_generation`  task: https://github.com/openvinotoolkit/openvino.genai/blob/23bf529766c52a05de074fd53637c321271d821b/tools/llm_bench/task/visual_language_generation.py#L185

Similar to `text_generation` task
https://github.com/openvinotoolkit/openvino.genai/blob/23bf529766c52a05de074fd53637c321271d821b/tools/llm_bench/task/text_generation.py#L218

Reply:
I don't got the question. User still can specify scheduler config for pipeline, I have deleted just default config here.